webrender/res/ps_radial_gradient.vs.glsl

```diff
@@ -27,7 +27,7 @@ void main(void) {
     vEndRadius = gradient.start_end_radius_extend_mode.y;
 
     // V coordinate of gradient row in lookup texture.
-    vGradientIndex = float(prim.sub_index) + 0.5;
+    vGradientIndex = float(prim.sub_index) * 2.0 + 0.5;
```

Comment:
I think we are overloading the `vGradientIndex` semantic a bit here, and the logic gets split between VS and PS too much. Perhaps, let's just do `vGradientIndex = prim.sub_idex` in the VS, and then do `vGradientIndex * 2.0 + 0.5 + 1.0/256.0` in PS? It would make no difference in terms of performance, but would be cleaner to read.